diff -aur datamash-1.7-orig/src/decorate.c datamash-1.7/src/decorate.c
--- datamash-1.7-orig/src/decorate.c	2022-02-21 16:32:01.390387464 +0100
+++ datamash-1.7/src/decorate.c	2022-02-21 16:46:36.966703253 +0100
@@ -442,7 +442,7 @@
         }
 
       if (!decorate_fields (&lb))
-        die (SORT_FAILURE, errno, _("conversion failed in line %zu"), linenum);
+        die (SORT_FAILURE, errno, _("conversion failed in line %lld"), linenum);
 
       if (debug)
         dbg_printf ("writing line: '%.*s'", (int)lb.length, lb.buffer);
